torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14490,6 +14490,21 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                # RuntimeError: attribute lookup is not defined on builtin
                DecorateInfo(unittest.expectedFailure, 'TestJit', 'test_variant_consistency_jit'),
            )),
+    OpInfo('chalf',
```

Comment:
See https://github.com/pytorch/pytorch/pull/75958 for an updated pattern

Reply:
Thanks!